### ISSUE TITLE
Emuhawk stops avi recording on core reboot

### ISSUE THREAD

**Issue by NN9VT** [TASEmulators/BizHawk#3654]
So I have some bk2 movie files, i want to make one AVI file from them. I play the first movie then load other movie in my order, but the AVI/WAV stops recording everytime i play other movies, it only record one movie.
![1111111](https://user-images.githubusercontent.com/130147477/236490681-25212a56-c668-46d7-979c-d7cb952dd130.png)

**Comment by YoshiRulz** [comment]
Pretty sure this is intended. Calling this a feature request I guess?

**Comment by NN9VT** [comment]
> 

yes, i'm using Boilsoft Video Joiner, it's very fast but it will take a some time especially when i have many files. It's ok but if i can shorten the process, it would be great!

**Comment by CasualPokePlayer** [comment]
This frankly won't be working the way you want usually, unless you force a constant size (or cores happen to have the same video size), and the cores happen to have the same vsync rate, a split will be required.

**Comment by vadosnaprimer** [comment]
If it splits on resolution change across core reboot, it would still be a nice feature since setting constant size is possible. And yeah if it also split on fps change, it'd be pretty usable IMO, because not always does the core have to change that from movie to movie.
